openshift/jenkins/Jenkinsfile.promote-to-test

```diff
@@ -0,0 +1,143 @@
+#!groovy
+
+// ------------------
+// Pipeline Variables
+// ------------------
+String[] DEPLOYMENT_ENVIRONMENT_TAGS = ['dev', 'test', 'prod']
+def cli
+def notify
+
+// --------------------
+// Declarative Pipeline
+// --------------------
+pipeline {
+  agent none
+  options {
+    disableResume()
+    buildDiscarder(logRotator(numToKeepStr: "5")) // keep 5 builds only
+  }
+  environment {
+    CI = "true"
+    PATHFINDER_URL = "pathfinder.gov.bc.ca"
+    VANITY_URL="https://pims-test.pathfinder.gov.bc.ca/"
+
+    // The name of your application
+    APP_NAME = "pims"
+
+    // Deployment configuration
+    DEV_ENV = "${DEPLOYMENT_ENVIRONMENT_TAGS[0]}"
+    TEST_ENV = "${DEPLOYMENT_ENVIRONMENT_TAGS[1]}"
+    PROD_ENV = "${DEPLOYMENT_ENVIRONMENT_TAGS[2]}"
+
+    // The name of the project namespace(s).
+    NAME_SPACE = "jcxjin"
+
+    // enable pipeline verbose debug output if greater than 0
+    DEBUG_OUTPUT = "false"
+  }
+  stages {
+    stage("Initialize") {
+      agent any
+      steps {
+        script {
+          // load supporting functions from external script files
+          cli = load "./openshift/jenkins/common.groovy"
+          notify = load "./openshift/jenkins/notify.groovy"
+
+          // ensure required secrets and credentials are available in the CI environment
+          // [add more secrets here if needed...]
+          sh "oc extract secret/rocket-chat-secrets --to=${env.WORKSPACE} --confirm"
+
+          // initialize CI environment values
+          env.ROCKET_DEPLOY_WEBHOOK = readFile("rocket-deploy-webhook").trim()
+
+          if (DEBUG_OUTPUT.equalsIgnoreCase("true")) {
+            // Force OpenShift Plugin directives to be verbose
+            openshift.logLevel(1)
+
+            // Print all environment variables
+            echo 'DEBUG - All pipeline environment variables:'
+            echo sh(returnStdout: true, script: 'env')
+          }
+        }
+      }
+    }
+    stage("Manual Approval") {
+      agent any
+      options {
+        timeout(time: 1, unit: "HOURS")
+      }
+      input {
+        message "Deploy to Staging/Test?"
+        ok "Yes, go ahead"
+      }
+      steps {
+        echo "[ci/cd] Promotion from DEV to STAGING (Test) was approved. Proceeding..."
+      }
+    }
+    stage("Promote API To Staging") {
```

Comment:
Should this be Test and not Staging? (one instance below as well).

Reply:
I will change it back to TEST. I wanted to avoid confusion with the name where it would imply automated tests being run